examples/aliases/aliases_example.py

```diff
@@ -22,3 +22,27 @@ class RunSettings:
 expected = """
 Namespace(train=RunSettings(debug=False, verbose=False), valid=RunSettings(debug=False, verbose=False))
 """
+
+parser.print_help()
+expected += """\
+usage: aliases_example.py [-h] [-train.d bool] [-train.v] [-valid.d bool] [-valid.v]
+
+optional arguments:
+  -h, --help            show this help message and exit
+
+RunSettings ['train']:
+  Parameters for a run.
+
+  -train.d bool, --train.debug bool, --notrain.debug bool
```

Comment:
This thing here is a little weird. I'd like `--train.nodebug`, not `--notrain.debug`.